calour/tests/test_io.py

```diff
@@ -54,6 +56,35 @@ def test_read(self):
         self.assertTrue(scipy.sparse.issparse(exp.data))
         self._validate_read(exp)
 
+    def test_read_openms_bucket_table(self):
+        # load the openms bucket table with no metadata
+        exp = ca.read(self.openms_csv, data_file_type='openms', sparse=False)
+        self.assertEqual(len(exp.sample_metadata), 9)
+        self.assertEqual(len(exp.feature_metadata), 10)
+        self.assertEqual(exp.shape, (9, 10))
+        self.assertEqual(exp.data[0, :].sum(), 8554202)
+        self.assertEqual(exp.data[:, 1].sum(), 13795540)
+        self.assertEqual(exp.sparse, False)
+
+    def test_read_open_ms(self):
+        exp = ca.read_open_ms(self.openms_csv)
+        # test we get the MZ and RT correct
+        self.assertIn('MZ', exp.feature_metadata)
+        self.assertIn('RT', exp.feature_metadata)
+        self.assertAlmostEqual(exp.feature_metadata['MZ'].iloc[1], 118.0869)
+        self.assertAlmostEqual(exp.feature_metadata['RT'].iloc[1], 23.9214)
```

Comment:
again i think loc takes value and not position? and we check the entries at row 1...